internal/dao/node.go

```diff
@@ -145,6 +146,8 @@ func (n *Node) List(ctx context.Context, ns string) ([]runtime.Object, error) {
 		nmx, _ = client.DialMetrics(n.Client()).FetchNodesMetricsMap(ctx)
 	}
 
+	shouldCountPods := os.Getenv("K9S_DISABLE_POD_COUNTING") != "true"
```

Comment:
Right not keen on putting env var. Best handled as a k9s config option as you've pointed out. 
We typically propagate these kind of configuration via the context that list takes as a param. 
Please take a peek at how we do this in the code ie dao.Alias and view.Alias as an example.

Reply:
Thank you for the pointer! https://github.com/derailed/k9s/pull/2168/commits/f2720688618bf89b6cf89ea3c3eec75b0560f780